axelor-sale/src/main/java/com/axelor/apps/sale/web/CartController.java

```diff
@@ -95,4 +100,23 @@ public void createSaleOrder(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void getCartPartnerDomain(ActionRequest request, ActionResponse response) {
+    try {
+      Cart cart = request.getContext().asType(Cart.class);
+      List<CartLine> cartLineList = cart.getCartLineList();
+      String domain =
+          Beans.get(SaleOrderDomainService.class).getPartnerBaseDomain(cart.getCompany());
+
+      if (!(cartLineList == null || cartLineList.isEmpty())) {
+        domain =
+            Beans.get(PricedOrderDomainService.class)
+                .getPartnerDomain(cart, domain, PriceListRepository.TYPE_SALE);
+      }
```

Comment:
I don't think we need this part of the domain since cart doesn't have a price list. We can just use SaleOrderDomainService#getPartnerBaseDomain.